system/aws_persistent_deployment.py

```diff
@@ -0,0 +1,90 @@
+import boto3
+from pprint import pprint
+from system.utils import PERSISTENT_INSTANCES, ORIGINAL_MAPPING, UPGRADE_MAPPING  # instance ids are the same for now
+import time
+import botocore
+
+
+REGION_NAMES = [
+    'us-east-1', 'us-east-2', 'us-west-1', 'us-west-2', 'ap-northeast-2', 'ap-southeast-1', 'ap-southeast-2',
+    'ap-northeast-1', 'ca-central-1', 'eu-central-1', 'eu-west-1', 'eu-west-2', 'sa-east-1'
+]
+DEVICE = '/dev/sda1'
+OWNER = '962610246670'
```

Comment:
It's better to add comment stating that normally this ID shouldn't change so if someone creates a PR which tries to trick this script to use some 3rd party snapshot it becomes immediately obvious

Reply:
Done.